src/app/containers/CpsMetadata/utils/getBrandedImage/index.js

```diff
@@ -0,0 +1,9 @@
+const getBrandedImage = (locator, service) => {
+  const iChefHost =
+    process.env.SIMORGH_APP_ENV === 'prod'
```

Comment:
You can use `isLive()` from `src/app/lib/utilities/isLive/`

Also `SIMORGH_APP_ENV` is one of `local|test|live` - not prod - thererfore this would always return false in this case.